jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/bigquery/BigQuerySqlDialect.java

```diff
@@ -0,0 +1,136 @@
+package com.exasol.adapter.dialects.bigquery;
+
+import static com.exasol.adapter.AdapterProperties.*;
+import static com.exasol.adapter.capabilities.AggregateFunctionCapability.*;
+import static com.exasol.adapter.capabilities.LiteralCapability.*;
+import static com.exasol.adapter.capabilities.MainCapability.*;
+import static com.exasol.adapter.capabilities.PredicateCapability.*;
+import static com.exasol.adapter.capabilities.ScalarFunctionCapability.*;
+
+import java.sql.*;
+import java.util.*;
+
+import com.exasol.adapter.*;
+import com.exasol.adapter.capabilities.*;
+import com.exasol.adapter.dialects.*;
+import com.exasol.adapter.jdbc.*;
+
+/**
+ * This class implement the SQL dialect of Google's Big Query
+ *
+ * @see <a href="https://aws.amazon.com/athena/">AWS Athena</a>
+ */
+public class BigQuerySqlDialect extends AbstractSqlDialect {
+    private static final String NAME = "BIG QUERY";
+    private static final Capabilities CAPABILITIES = createCapabilityList();
+    private static final List<String> SUPPORTED_PROPERTIES = Arrays.asList(SQL_DIALECT_PROPERTY,
+            CONNECTION_NAME_PROPERTY, CONNECTION_STRING_PROPERTY, USERNAME_PROPERTY, PASSWORD_PROPERTY,
+            CATALOG_NAME_PROPERTY, SCHEMA_NAME_PROPERTY, TABLE_FILTER_PROPERTY, EXCLUDED_CAPABILITIES_PROPERTY,
+            DEBUG_ADDRESS_PROPERTY, LOG_LEVEL_PROPERTY);
```

Comment:
I am thinking whether we should automatically set some of those in the base dialect since the dialect can't decide not to support them (e.g. `SQL_DIALECT_PROPERTY`, `DEBUG_ADDRESS_PROPERTY`, `LOG_LEVEL_PROPERTY`).

Reply:
created ticket